index.tsx

```diff
@@ -510,6 +520,18 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
     this.flatlistRef.current._component.scrollToOffset({ offset })
   })
 
+  scrollToIndex = (params: {animated: boolean, index: number, viewOffset: number, viewPosition: number}): void => {
+    this.flatlistRef.current._component.scrollToIndex(params)
+  }
+
+  scrollToEnd = (params: {animated: boolean}): void => {
+    this.flatlistRef.current._component.scrollToEnd(params)
+  }	
+    
+  getItemLayout = (func: GetItemLayoutFunction ): void => {
```

Comment:
hm that's not quite what i meant. Props are spread onto the underlying flatlist. is there a reason that passing `getItemLayout` as a prop doesn't work, like in a normal flatlist? https://github.com/computerjazz/react-native-draggable-flatlist/blob/master/index.tsx#L802

Reply:
oh sorry, now I get what you mean. 
Anyway I checked the code that was using getItemLayout and I think it was legacy code not useful anymore. So I removed that part.